coremltools/converters/mil/frontend/tensorflow/test/test_ops.py

```diff
@@ -2105,6 +2105,13 @@ def test_unary(self, use_cpu_for_conversion, backend, rank, mode):
         if not use_cpu_only and mode in self._FP16_UNSUPPORTED:
             return
 
+        if _macos_version() < (13, 0):
+            if not use_cpu_for_conversion:
+                pytest.xfail("GPU issue fixed in iOS16/macOS13")
```

Comment:
I would suggest just use `pytest.skip`,
cause we are not going to fix this anyway.

In the future, when we find the pattern `pytest.xfail`, that can correctly reflect how many real issues are we going to fix